frontend/package.json

```diff
@@ -108,6 +108,8 @@
     "react-device-detect": "1.9.9",
     "react-diff-viewer-continued": "^3.3.1",
     "react-dom": "16.14.0",
+    "react-chartjs-2": "^3.0.3",
+    "chart.js": "^3.5.1",
```

Comment:
We already have recharts, I don't think adding another chart library is a good idea.

Reply:
will delete these two dependencies and switch to recharts